src/components/_data/ContentSingularData.ts

```diff
@@ -1,6 +1,9 @@
+import { ToDoListItem } from '../ContentWidgets/ToDoList/ToDoList';
+
 export type ContentSingularData = {
     plainText_content?: string
     exampleImage_imageLink?: string
     exampleImage_percentageSize?: number
+    toDoList_content?: ToDoListItem[]
```

Comment:
Ah so this is the issue of your method of ordering. The way I did my ordering was I had `content` *and* `order` at the same level. You have `order` nested within `content`. What I would do is have right here `toDoList_content` and `toDoList_order`.